airbyte-integrations/connectors/destination-teradata/src/main/java/io/airbyte/integrations/destination/teradata/TeradataDestination.java

```diff
@@ -69,7 +73,8 @@ protected Map<String, String> getDefaultConnectionProperties(final JsonNode conf
             } else {
                 additionalParameters.put(PARAM_SSLMODE, REQUIRE);
             }
-        }
+        } 
+       	additionalParameters.put(ENCRYPTDATA, ENCRYPTDATA_ON);
```

Comment:
Does this need to be removed for TLS connections, or is it OK to always have this option present?

Reply:
It is okay to always have this option @evantahler